### ISSUE TITLE
Problem with latest release when project has no assets

### ISSUE THREAD

**Issue by jduthon** [electrode-io/electrode-native#1064]
With the latest release (`0.29.3`), when running `ern run-ios` on our project it fails and I get the following error:
```
error: MY_HOME/.ern/containergen/out/ios/ElectrodeContainer/Libraries/MiniApp/assets: No such file or directory
```

**Comment by belemaire** [comment]
@jduthon Fixed in 0.29.4.
Sorry for the delay.

**Comment by jduthon** [comment]
@belemaire well that was actually pretty fast :) 
Thanks a lot!
